unity-renderer/Assets/Scripts/MainScripts/DCL/Controllers/ExploreV2/Scripts/Sections/PlacesAndEventsSection/PlacesAndEventsCardsFactory.cs

```diff
@@ -0,0 +1,97 @@
+
+public static class PlacesAndEventsCardsFactory
+{
+    internal const string EVENT_CARD_MODAL_ID = "EventCard_Modal";
+    internal const string PLACE_CARD_MODAL_ID = "PlaceCard_Modal";
+
+    /// <summary>
+    /// Creates and configures a pool for cards.
+    /// </summary>
+    /// <param name="pool">Pool to configure.</param>
+    /// <param name="poolName">Name of the pool.</param>
+    /// <param name="cardPrefab">Card prefab to use by the pool.</param>
+    /// <param name="maxPrewarmCount">Max number of pre-created cards.</param>
+    public static void GetCardsPoolLazy(out Pool pool, string poolName, BaseComponentView cardPrefab, int maxPrewarmCount)
+    {
+        pool = PoolManager.i.GetPool(poolName);
+
+        if (pool != null) return;
+
+        pool = PoolManager.i.AddPool(poolName, Object.Instantiate(cardPrefab).gameObject, maxPrewarmCount: maxPrewarmCount, isPersistent: true);
+        pool.ForcePrewarm();
+    }
+
+    public static EventCardComponentView CreateConfiguredEventCard(Pool pool, EventCardComponentModel eventInfo, Action<EventCardComponentModel> OnEventInfoClicked, Action<EventFromAPIModel> OnEventJumpInClicked, Action<string> OnEventSubscribeEventClicked, Action<string> OnEventUnsubscribeEventClicked) =>
+        EventsCardsConfigurator.Configure(pool.Get<EventCardComponentView>(), eventInfo, OnEventInfoClicked, OnEventJumpInClicked, OnEventSubscribeEventClicked, OnEventUnsubscribeEventClicked);
+
+    public static PlaceCardComponentView CreateConfiguredPlaceCard(Pool pool, PlaceCardComponentModel placeInfo, Action<PlaceCardComponentModel> OnPlaceInfoClicked, Action<HotSceneInfo> OnPlaceJumpInClicked) =>
+        PlacesCardsConfigurator.Configure(pool.Get<PlaceCardComponentView>(), placeInfo, OnPlaceInfoClicked, OnPlaceJumpInClicked);
+
+    /// <summary>
+    /// Instantiates (if does not already exists) a place card modal from the given prefab.
+    /// </summary>
+    /// <param name="placeCardModalPrefab">Prefab to instantiate.</param>
+    /// <returns>An instance of a place card modal.</returns>
+    public static PlaceCardComponentView GetPlaceCardTemplateHiddenLazy(PlaceCardComponentView placeCardModalPrefab) =>
+        GetCardTemplateHiddenLazy(placeCardModalPrefab, PLACE_CARD_MODAL_ID);
+
+    /// <summary>
+    /// Instantiates (if does not already exists) a event card modal from the given prefab.
+    /// </summary>
+    /// <param name="eventCardModalPrefab">Prefab to instantiate.</param>
+    /// <returns>An instance of a event card modal.</returns>
+    public static EventCardComponentView GetEventCardTemplateHiddenLazy(EventCardComponentView eventCardModalPrefab) =>
+        GetCardTemplateHiddenLazy(eventCardModalPrefab, EVENT_CARD_MODAL_ID);
+
+    private static TCardView GetCardTemplateHiddenLazy<TCardView>(TCardView modalPrefab, string cardModalId) where TCardView: BaseComponentView
+    {
+        TCardView modal;
+
+        GameObject existingModal = GameObject.Find(cardModalId);
```

Comment:
Isn't this `GameObject.Find` super slow? what's the idea? Shouldnt a Pool do this?

Reply:
It is. I also saw this issue while refactoring and felt it as a dirty solution. But I left it for 2 reasons 
- this function called only 3 times - once per each `Start` of `Places/Events/Highlights..Controller`s respectively. So we will not win so much with it.
- creating a variable to store it would require converting this Factory class to not-static and injecting it in Controllers (probably via ServiceLocator). This sounds good, but would require a bit more time ( reminding than this was just a side refactoring during the bug fix :) ). I just didn't want that this task grew up too much in time. But I can give it a try... Do you think I should go for it?

Reply:
But I don't have full context of what and why this template card exist and why it even should be instantiated and sometimes shown... I think @sandrade-dcl can bring us more context for it

Reply:
If it was originally there we can keep it, it's ugly but let's not increase the scope of this refactor without a good reason.